index.html

```diff
@@ -29,23 +34,27 @@ <h1>ROU<span class="green">LETTER</span></h1>
 		<div class="divider"></div>
 		<div class="grid"></div>
 		<div class="display">
-			<div class="wrapper">SCORE: 
+			<div class="wrapper">SCORE:
 				<span class="green">
 					<span id="score-value">0</span>
 				</span>
 			</div>
-			<div class="wrapper">BANK: 
+			<div class="wrapper">BANK:
 				<span class="green">
 					$<span id="bank-value">0</span>
 				</span>
 			</div>
 		</div>
 		<div class="shop"></div>
-		<button class="refresh">REFRESH</button>
+		<div>
+			<button class="shop-button refresh-button">REFRESH</button>
+			<button class="shop-button shuffle-button">🔀</button>
```

Comment:
Can we change the emoji to just the word shuffle and smaller text? I'll update it with a background image later